test/setup.ts

```diff
@@ -58,8 +58,8 @@ const setupTest = deployments.createFixture<
 });
 
 export const setupImpactScopes = async (
-  contract: HypercertMinterV0,
-  contractAtAddress?: HypercertMinterV0,
+  contract: HypercertMinterV1 | HypercertMinterV0,
```

Comment:
Hmm this looks a bit of a PITA every time we do an upgrade - maybe worth defining a union type somewhere and then we just have one place to update?